### PR TITLE
Fix #167 - Show instructions on configuring your editor

### LINKED ISSUE #167: Configure your editor
I just went through the Elm tour (https://guide.elm-lang.org/install.html) and saw this:

image

I think this is really helpful and we should include sth. like this as well.

### PR DESCRIPTION
**Pull request by wilzbach**
Some editors plugins might be a bit controversial, but overall my hope is to point readers quickly to their favorite editor.

### CODE REVIEW

File: welcome/install-d-locally.md [hunk 1]
```diff
@@ -43,3 +43,24 @@ for other platforms like e.g. ARM.
...
+- [IntelliJ](https://github.com/kingsleyh/DLanguage)
+- [Mono](https://wiki.dlang.org/Mono-D)
+- [Sublime Text](https://github.com/yazd/DKit)
+- [Vim](https://wiki.dlang.org/D_in_Vim)
+- [VS Code](https://marketplace.visualstudio.com/items/webfreak.code-d)
+- [Visual Studio](http://rainers.github.io/visuald/visuald/StartPage.html)
+
+You may also want to try an IDE written in D for D:
+
+- [Coedit](https://github.com/BBasile/Coedit)
```
Comment (ZombineDev) [inline welcome/install-d-locally.md]: Coedit is written in Pascal, but on the other hand I think it is the most mature and feature-full (as in D specific features) IDE for D, though I have never tried as I prefer VIM and VS Code VIM mode

File: welcome/install-d-locally.md [hunk 2]
```diff
@@ -43,3 +43,24 @@ for other platforms like e.g. ARM.
...
+## Configure your editor
+
+Using D is way nicer when you have a code editor or IDE to help you out.
+There are D plugins for at least the following editors:
+
+- [Atom](https://github.com/Pure-D/atomize-d)
+- [Eclipse](http://ddt-ide.github.io)
+- [Emacs](https://github.com/Emacs-D-Mode-Maintainers/Emacs-D-Mode)
+- [IntelliJ](https://github.com/kingsleyh/DLanguage)
+- [Mono](https://wiki.dlang.org/Mono-D)
```
Comment (ZombineDev) [inline welcome/install-d-locally.md]: I'm not sure if we should advertise Mono-D as it has been unmaintained for almost an year (specifically it doesn't work with newer versions of MonoDevelop / Xamarin Studio / Visual Studio for Mac). That said, it was one of the best D IDEs 2-3 years ago - I think it was the first D IDE to have native dub support and go to definition, even before DCD existed.

**Comment by wilzbach** [comment]
This seems to be a common enough question, e.g. https://stackoverflow.com/questions/45282557/d-digital-mars-compiler-issue-w-intellj

**Comment by wilzbach** [comment]
(rebased & addressed comments)

**Pull request merged**